src/Discord/Discord.php

```diff
@@ -290,6 +290,51 @@ class Discord
      */
     protected $client;
 
+    /**
+     * The HTML Color Table.
+     *
+     * @array HTML Color Table.
+     */
+    public $colorTable = [
+        'indianred' => 'cd5c5c', 'lightcoral' => 'f08080', 'salmon' => 'fa8072', 'darksalmon' => 'e9967a',
```

Comment:
Might also consider storing the values as their hex encoded value and save the string->int conversion via ```hexdec()``` on each use.

You'd then have ```'indianred' => 0xcd5c5c, ...```, and in the ```getColor()``` function you'd simply return ```$this->colorTable[$colorName]``` for found keys.

Reply:
will do that now

Reply:
fd320c2 contains @kjc75's sugestion
69649cb contains @davidcole1340's change request